### PR TITLE
Specify environment module path for bolt

### PR DESCRIPTION
**Pull request by traylenator**
#### Pull Request (PR) description

Somewhere between voxpupuli-acceptance last v1 and v2 the modules are installed in a path where bolt can't find them.

#### This Pull Request (PR) fixes the following issues
Fixes #474 
Fixes #472

### CODE REVIEW

**Comment by traylenator** [comment]
Has to be be a better way of getting the environment than hard-coding it.

File: spec/acceptance/redis_cli_task_spec.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 describe 'redis-cli task' do
   subject do
-    on(default, "bolt task run --modulepath /etc/puppetlabs/code/modules --targets localhost #{task_name} #{params}", acceptable_exit_codes: [0, 1]).stdout
+    on(default, "bolt task run --modulepath /etc/puppetlabs/code/environments/production/modules --targets localhost #{task_name} #{params}", acceptable_exit_codes: [0, 1]).stdout
```
Comment (ekohl) [inline spec/acceptance/redis_cli_task_spec.rb:7]: Is it still needed to specify the modulepath at all? Isn't this the default built in?

File: spec/acceptance/suites/scl/redis5_spec.rb [hunk 2]
```diff
@@ -4,11 +4,11 @@
 
 describe 'redis', if: %w[centos redhat].include?(os[:family]) && os[:release].to_i == 7 do
   before(:all) do
-    on hosts, puppet_resource('service', 'redis', 'ensure=stopped', 'enable=false')
+    apply_manifest('service{"redis" : ensure => stopped, enable => false}')
```
Comment (ekohl) [inline spec/acceptance/suites/scl/redis5_spec.rb:7]: A closer match (in multi host systems):
```suggestion
    apply_manifest_on(hosts, 'service{"redis" : ensure => stopped, enable => false}')
```

Or keeping it closer to the previous code I think this should still work:
```suggestion
    on hosts, puppet('resource', 'service', 'redis', 'ensure=stopped', 'enable=false')
```

File: spec/acceptance/redis_cli_task_spec.rb (hunk 1 above)
Comment (traylenator) [reply spec/acceptance/redis_cli_task_spec.rb:7]: Default was `/root/.puppetlabs` unfortunately.

Comment (ekohl) [reply spec/acceptance/redis_cli_task_spec.rb:7]: Perhaps works, and let Puppet interpolate it. Not sure how much escaping is needed.
```suggestion
    on(default, "bolt task run --modulepath '/etc/puppetlabs/code/environments/$envirornment/modules' --targets localhost #{task_name} #{params}", acceptable_exit_codes: [0, 1]).stdout
```

Comment (traylenator) [reply spec/acceptance/redis_cli_task_spec.rb:7]: Tried various level of escaping and brackets with `$environment`

Tried `%x(puppet config print modulepath)` in various ways

**Pull request merged**